src/ngx_http_lua_ssl_session_fetchby.c

```diff
@@ -307,8 +307,12 @@ ngx_http_lua_ssl_sess_fetch_handler(ngx_ssl_conn_t *ssl_conn, u_char *id,
     L = ngx_http_lua_get_lua_vm(r, NULL);
 
     c->log->action = "fetching SSL session by lua";
-
-    rc = lscf->srv.ssl_sess_fetch_handler(r, lscf, L);
+    
+    if(lscf->srv.ssl_sess_fetch_handler) {
```

Comment:
Style: need a space after `if`. Also, we should test the pointer value against `NULL` instead of testing it as a boolean as per nginx's coding convention.